src/DataTypes/Serializations/SerializationBool.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <DataTypes/Serializations/SerializationCustomSimpleText.h>
+
+namespace DB
+{
+
+class SerializationBool final : public SerializationCustomSimpleText
```

Comment:
We also have format `TSVRaw` that uses methods `deserializeTextRaw/serializeTextRaw` from `ISerialization`, we should serialize/deserialize boolean in this format the same way as in `TSV`, let's add these methods to `SerializationBool`.

Reply:
ok ,i will do this